src/main/kotlin/racingcar/view/OutputView.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.view
+
+import racingcar.view.output.Output
+
+class OutputView(private val output: Output) {
+
+    fun show(movingDistances: List<Int>) {
+        val message = movingDistances.joinToString("") { "${this.statusBy(it)}\n" }
+        output.show(message)
+    }
+
+    fun statusBy(movingDistance: Int): String {
+        return (0 until movingDistance).joinToString("") { "-" }
```

Comment:
`String`에 대해 호출할 수 있는 `repeat()`가 있습니다.

```suggestion
        return "-".repeat(movingDistance)
```